apps/domain-admin/1.6.62/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     volumes:
       - "./data/database:/app/database"
       - "./data/logs:/app/logs"
-    image: mouday/domain-admin:v1.6.62
+    image: mouday/domain-admin:v1.6.63
     labels:  
       createdBy: "Apps"
 networks:  
```

Comment:
The only significant difference between the two images is that v1.6.62 was replaced with v1.6.63 in the `image` field of the Docker Compose file. This change may result in changes to the version-specific configuration and features of the application running under this service. It's recommended to review any relevant documentation or support information for the new version to ensure compatibility with existing configurations and avoid potential errors.